lib/next_ls/db.ex

```diff
@@ -85,9 +85,9 @@ defmodule NextLS.DB do
       {conn, s.logger},
       ~Q"""
       DELETE FROM symbols
-      WHERE module = ?;
+      WHERE module = ? OR file = ?;
```

Comment:
I think this might need to be `AND`, because if two modules are written in the same file, either by nested `defmodule` or just two root `defmodule`s, the query written with the `OR` will clobber each other and delete symbols for another module.

But, I don't think that would solve the problem you are attempting to solve